src/measurement_buffer.cpp

```diff
@@ -152,7 +152,21 @@ void MeasurementBuffer::BufferPCLCloud(const \
 
     point_cloud_ptr cld_global(new pcl::PointCloud<pcl::PointXYZ>);
 
-    _buffer.transform(cloud, *cld_global, _global_frame);
+    // Create a tf::Transform using the tfStamped for _buffer
+    tf_stamped = _buffer.lookupTransform(_global_frame, cloud.header.frame_id.c_str(), ros::Time(0));
+    tf::Quaternion q(tf_stamped.transform.rotation.x,
+                     tf_stamped.transform.rotation.y,
+                     tf_stamped.transform.rotation.z,
+                     tf_stamped.transform.rotation.w);
+
+    tf::Vector3 t(tf_stamped.transform.translation.x,
+                  tf_stamped.transform.translation.y,
+                  tf_stamped.transform.translation.z);
+
+    tf::Transform transform(q, t);
```

Comment:
This can be streamlined using TF's built in conversion options: http://docs.ros.org/melodic/api/tf/html/c++/transform__datatypes_8h.html

You can use ```tf::transformMsgToTF (const geometry_msgs::Transform &msg, Transform &bt)``` to directly go from geometry_msgs::Transform to tf::Transform in 1 line. Would change this section to:

```
tf::Transform tf_transform;
tf_stamped = _buffer.lookupTransform(_global_frame, cloud.header.frame_id.c_str(), ros::Time(0));
tf::transformMsgToTF(tf_stamped, tf_transform)
```